DeploymentGuide.md

```diff
@@ -4,35 +4,46 @@ Management Server component of uProxy for Orgs (UfO)
 
 ## Deployment
 
-### Create an Instance
+### Deploy an Instance
```

Comment:
That's denoting a section which should be different from the title. I made the title Deployment Guide (since that's more fitting anyway) which should help.